lib/ruby_ui/theme_toggle/set_dark_mode.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module RubyUI
+  class SetDarkMode < ThemeToggle
```

Comment:
I don't like applying inheritance on components, and I think it should inherits from Base since ThemeToggle doesn't have any relevant behavior.
```suggestion
  class SetDarkMode < Base
```

Reply:
Yeah good point. I merged it before your review. Im fixing it now.

Reply:
No problem, I just noticed it after the review 😅